src/app/containers/ATIAnalytics/index.test.jsx

```diff
@@ -211,14 +215,14 @@ describe('ATI Analytics Container', () => {
       canonical.default = mockCanonical;
 
       render(
-        <ContextWrap platform="canonical" pageType={STORY_PAGE} service="news">
+        <ContextWrap platform="canonical" pageType={STORY_PAGE} service="mundo">
```

Comment:
What are these changes for?

Reply:
Because the fixture data in the tests didn't match the service that the test was told it was, so the code was trying to run news-specific logic on an asset that was from mundo... which naturally failed